misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/OptionalAssertionsSpec.kt

```diff
@@ -42,4 +43,24 @@ abstract class OptionalAssertionsSpec (
             }
         }
     }
+
+    describeFun(isPresent.name) {
+        val isPresentFun = isPresent.lambda
+
+        context("is present") {
+            it("does not throw") {
+                val isPresentValue: Optional<Any> = Optional.of(2)
```

Comment:
I prefer your name `isPresentValue` over the existing `nonEmptyValue` so I suggest you use your name when you move this above.

Reply:
I think both val make sense depending on the test. Let me know if you still want to use the same in the upcoming code.